doc/classes/Basis.xml

```diff
@@ -340,94 +374,127 @@
 	</methods>
 	<members>
 		<member name="x" type="Vector3" setter="" getter="" default="Vector3(1, 0, 0)">
-			The basis matrix's X vector (column 0). Equivalent to array index [code]0[/code].
+			The basis's X axis, and the column [code]0[/code] of the matrix.
+			On the identity basis, this vector points right ([constant Vector3.RIGHT]).
 		</member>
 		<member name="y" type="Vector3" setter="" getter="" default="Vector3(0, 1, 0)">
-			The basis matrix's Y vector (column 1). Equivalent to array index [code]1[/code].
+			The basis's Y axis, and the column [code]1[/code] of the matrix.
+			On the identity basis, this vector points up ([constant Vector3.UP]).
 		</member>
 		<member name="z" type="Vector3" setter="" getter="" default="Vector3(0, 0, 1)">
-			The basis matrix's Z vector (column 2). Equivalent to array index [code]2[/code].
+			The basis's Z axis, and the column [code]2[/code] of the matrix.
+			On the identity basis, this vector points back ([constant Vector3.BACK]).
 		</member>
 	</members>
 	<constants>
 		<constant name="IDENTITY" value="Basis(1, 0, 0, 0, 1, 0, 0, 0, 1)">
-			The identity basis, with no rotation or scaling applied.
+			The identity basis. This is a basis with no rotation, no shear, and its scale being [code]1[/code]. This means that:
+			- The [member x] points right ([constant Vector3.RIGHT]);
+			- The [member y] points up ([constant Vector3.UP]);
+			- The [member z] points back ([constant Vector3.BACK]).
+			[codeblock]
+			var basis := Basis.IDENTITY
+			print("| X | Y | Z")
+			print("| %s | %s | %s" % [basis.x.x, basis.y.x, basis.z.x])
+			print("| %s | %s | %s" % [basis.x.y, basis.y.y, basis.z.y])
+			print("| %s | %s | %s" % [basis.x.z, basis.y.z, basis.z.z])
+			# Prints:
+			# | X | Y | Z
+			# | 1 | 0 | 0
+			# | 0 | 1 | 0
+			# | 0 | 0 | 1
+			[/codeblock]
 			This is identical to creating [constructor Basis] without any parameters. This constant can be used to make your code clearer, and for consistency with C#.
 		</constant>
 		<constant name="FLIP_X" value="Basis(-1, 0, 0, 0, 1, 0, 0, 0, 1)">
-			The basis that will flip something along the X axis when used in a transformation.
+			When any basis is transformed (multiplied) by [constant FLIP_X], it will be flipped along its [member x] axis. This negates all [member x]'s components.
+			If [constant FLIP_X] is transformed by any other basis, the [member Vector3.x] component of all axes will be negated, instead.
```

Comment:
The wording here seems a bit weird, conceptually speaking. When we multiply the order is `parent * child`. We probably would say that child is transformed by parent, not parent is transformed by child, but really the result is a composition of both matrices, so perhaps we can just say multiplied.

The statement "it will be flipped along its [member x] axis" is not correct. It doesn't flip the whole basis along the axis, it just flips the axis. Proof:

```gdscript
func _ready():
	var b := Basis.IDENTITY
	b.y = Vector3(1, 1, 0)
	print(b * Basis.FLIP_X)
	# Output: [X: (-1, 0, 0), Y: (1, 1, 0), Z: (0, 0, 1)]
```

```suggestion
			When any basis is multiplied by [constant FLIP_X], it negates all components of the [member x] axis (the X column).
			When [constant FLIP_X] is multiplied by any basis, it negates the [member Vector3.x] component of all axes (the X row).
```